instance/models/openedx_instance.py

```diff
@@ -279,6 +279,11 @@ def get_load_balancer_configuration(self):
             # The active appserver doesn't have a public IP address.  This means that the server
             # has been terminated, so we don't show the preliminary page in this case, and simply
             # deconfigure the backend instead.
+            if triggered_by_instance:
+                self.logger.info(
```

Comment:
@antoviaque Yep, updated.  I also removed the `if triggered_by_instance` to make sure this gets always logged, since it shouldn't be happening.